city_scrapers/spiders/il_police_professionalism.py

```diff
@@ -0,0 +1,89 @@
+
+from city_scrapers_core.constants import (COMMISSION)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlPoliceProfessionalismSpider(CityScrapersSpider):
+    name = "il_police_professionalism"
+    agency = "Illinois Commission on Police Professionalism"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.isp.state.il.us"]
+    start_urls = ["https://www.isp.state.il.us/media/openmtgs.cfm"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.xpath("//*[contains(text(),\"Police Professionalism\")]"):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Commission on Police Professionalism"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return COMMISSION
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        title = item.xpath('text()').get()
+        date = "".join(title[title.index('(') + 1:title.index(')')].split(',')[1:])
+        date = date.strip(' ')
+        date = date.replace('p.m.', 'PM')
+        date = date.replace('a.m.', 'AM')
+        finalDate = datetime.strptime(date, "%B %d %Y %H:%M %p")
+        return finalDate
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        return {
+            "address": "",
+            "name": "Commission on Police Professionalism",
+        }
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        link = item.xpath('@href').get()
+        return [{
+            "href": "https://www.isp.state.il.us/media/" + link,
```

Comment:
It's generally easier to use `response.urljoin` to create the URL that would result from clicking the link on the page than statically adding the base path

Reply:
Cool! Thanks for the suggestion